apps/frontend/src/pages/settings/billing/index.vue

```diff
@@ -626,12 +626,12 @@ import {
 	OverflowMenu,
 	PurchaseModal,
 } from '@modrinth/ui'
+import ServerListing from '@modrinth/ui/src/components/servers/ServerListing.vue'
```

Comment:
shouldnt this be in with the other modrinth/ui imports rather than relying on a path in the library?